virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,73 @@
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+
+Manage a virtual machine.
+
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Network options:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[Configure a Linux bridge network], in addition to the default pod network.
+* Install the xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[SR-IOV Operator], configure a xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[SR-IOV network device], and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[attach a virtual machine to the SR-IOV network].
+
+Microsoft Windows virtual machine options:
+
+* xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows virtual machine installations with `sysprep`].
+* Connect to a Microsoft Windows virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-rdp-console_virt-accessing-vm-consoles[RDP].
+
+Monitoring {VirtProductName}
+
+* Monitor virtualization resources, details, status, and top consumers on the  Review the xref:../virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc#virt-reviewing-virtualization-overview[*Virtualization Overview* page].
+* View high-level information about your virtual machines on the xref:../virt/logging_events_monitoring/virt-viewing-information-about-vm-workloads.adoc#virt-viewing-information-about-vm-workloads[*Virtual Machines* page.
+* View virtual machine xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs].
+
+[id="virt-getting-started-additional-resources"]
+== Additional resources
+
+* xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines]
+* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration]
+* xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-vm-template[Virtual machine templates]
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage]
+* xref:../virt/virtual_machines/advanced_vm_management/virt-automating-management-tasks.adoc#virt-automating-virtual-machine-creation-with-ansible_virt-automating-management-tasks[Automating deployments] with Ansible.
```

Comment:
It looks odd to have only this list item end with a period. Suggest you remove the period and put the entire text in the link for consistency.